### PR TITLE
Nested nav

### PR DESCRIPTION
**Pull request by athnme**
So, I tried adding react navigation and nested multiple navigation stacks into each other to give this whole thing some hierarchical structure. I also tried to style the nav-bars as best as I could. Last but not least I refactored all the stacks so the code gets a bit more legible.

I would to hear your thoughts and critiques on this.

### CODE REVIEW

**Comment by athnme** [comment]
Ahhhh,, typos! 
What I have meant to say is: 
I would LOVE to hear your thoughts and critiques on this.

File: components/BottomNav.js [hunk 1]
```diff
@@ -83,7 +34,7 @@ export default function BottomNav() {
         inactiveColor="#cecece"
         shifting="true"
         labeled="true"
-        barStyle={{ backgroundColor: "#030D12" }}
+        barStyle={{ backgroundColor: "#030D12", height: 56 }}
```
Comment (bwnstck) [inline components/BottomNav.js:37]: is it possible to use variables for css colors? if yes i would suggest to use them :)
```suggestion
        barStyle={{ backgroundColor: var(--your-color), height: 56 }}
```

**Review by bwnstck — APPROVED** [review]
lgtm 🦄 as far as I can judge :)

**Comment by Sina-Sche** [comment]
Looking good as far as I can tell ;) 🏆 for using ReactNative!

**Pull request merged**